src/utils/auth.js

```diff
@@ -0,0 +1,11 @@
+let tokenRequest = '';
+
+const retToken = () => tokenRequest;
+
+const authParse = (req, res, next) => {
+  const rawToken = req.get('authorization');
+  tokenRequest = rawToken;
```

Comment:
I think it could be problematic once you get two requests from distinct tenants, the global variable will be overridden in this way causing a misuse of the token